### PR TITLE
Better documentation for --table_path option

### LINKED ISSUE #1590: File-based benchmark: --table_path should be better documented
For outsiders, it is unclear what they should put into the table_path. CREATE statements? CSVs?

### PR DESCRIPTION
**Pull request by mrzzzrm**
Fix #1590

### CODE REVIEW

File: src/benchmark/file_based_benchmark.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ int main(int argc, char* argv[]) {
 
   // clang-format off
   cli_options.add_options()
-      ("table_path", "Directory containing the Tables", cxxopts::value<std::string>()->default_value("")) // NOLINT
+      ("table_path", "Directory containing the Tables as csv, tbl or binary files", cxxopts::value<std::string>()->default_value("")) // NOLINT
```
Comment (mrks) [inline src/benchmark/file_based_benchmark.cpp]: CSVs require meta files, don't they?

Comment (mrzzzrm) [reply src/benchmark/file_based_benchmark.cpp]: Yes. I will try to document them as thoroughly as possible in the one line that they get in `--help`.

File: src/lib/utils/check_table_equal.cpp [hunk 2]
```diff
@@ -18,7 +18,7 @@ using namespace opossum;  // NOLINT
 using Matrix = std::vector<std::vector<AllTypeVariant>>;
 
 Matrix table_to_matrix(const std::shared_ptr<const Table>& table) {
-  // initialize matrix with table sizes, including column names/types
+  // initialize matrix with table sizes, including column names/typesF
```
Comment (mrks) [inline src/lib/utils/check_table_equal.cpp]: ?

File: src/benchmark/file_based_benchmark.cpp [hunk 3]
```diff
@@ -23,8 +23,8 @@ int main(int argc, char* argv[]) {
 
   // clang-format off
   cli_options.add_options()
-      ("table_path", "Directory containing the Tables", cxxopts::value<std::string>()->default_value("")) // NOLINT
-      ("query_path", "Directory/file containing the queries", cxxopts::value<std::string>()->default_value("")) // NOLINT
+      ("table_path", "Directory containing the Tables as csv, tbl or binary files. CSV files require meta-files, see csv_meta.hpp or any *.csv.json file.", cxxopts::value<std::string>()->default_value(".")) // NOLINT
+      ("query_path", "Directory containing .sql files or a specific .sql file", cxxopts::value<std::string>()->default_value(".")) // NOLINT
```
Comment (mrks) [inline src/benchmark/file_based_benchmark.cpp]: Mmh, "directory containing ... a specific file"? Maybe flip to remove ambiguity? "Single file or dir"

**Pull request merged**